src/main/asciidoc/object-mapping.adoc

```diff
@@ -17,10 +17,10 @@ This means we need two fundamental steps:
 Spring Data automatically tries to detect a persistent entity's constructor to be used to materialize objects of that type.
 The resolution algorithm works as follows:
 
-1. If there's a no-argument constructor, it will be used.
+1. If there is a single constructor, it will be used.
```

Comment:
In all three lines, change "will be" to "is"

---------

This is a general rule. The idea is to keep the tense consistent, so we use the indefinite present (also called the simple present) tense.

Reply:
Thanks for the explanation. I think the use of "will be" is a wrong translation of passive voice which is required German.